vip_spec.xsd

```diff
@@ -579,6 +580,30 @@ reference elements for polling location
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:simpleType name="DistrictType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="city" />
+      <xs:enumeration value="congressional" />
+      <xs:enumeration value="council" />
```

Comment:
> we would be unable to serve the VIP data even if it includes unique contests.

Hmm, I thought I asked basically this same question before [here](https://github.com/votinginfoproject/vip-specification/issues/30#issuecomment-93998098), and @jdmgoogle replied that it was okay.  He replied--

> No, there's no problem if a precinct links to more than one electoral district of the same type.